src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.model;
+
+public class Fruit {
```

Comment:
```suggestion
public class FruitTransaction {
```
Fruit - is a name for an entity/document (You will learn it later). This class isn't both of them.